src/components/EditTodo.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+const EditTodo = (props) => {
+    return (
+        <>
+            <h1>Todo App</h1>
+            <div className="showItask">
+                {
+                    props.itask.map((elem) => {
```

Comment:
this component's code looks like it should be TodoList component. @shivam-innotech